### DISCUSSION TITLE
Cannot proceed with payment github copilot

### DISCUSSION THREAD

**Discussion by FireFront1** [community/community#31013] [Copilot Conversations]
I tried to pay for github copilot, I had a problem with the card, and today I go in and it says to me that the ability to pay is not available !!! What does it mean??

**Comment by davecheney** [comment]
@FireWookie i'm sorry you're having difficulties, please contact support, https://support.github.com/contact

**Comment by thomas-ayissi** [comment]
Hello,

## Bank balance and authorisations for your bank card
1/ check that you have enough money in your account.
2/ also check with your bank that your payment method allows you to make online purchases.
Theoretically, if you have enough money on your account and if your card allows you to buy on the web, then you should be able to take a subscription for GitHub Co Pilot.

## Are your details correct on GitHub?
If you still can't get a subscription for Github Co-pilot, then check that your card details are correct.
Click on "settings" in the top right corner.
See the "payment" section and check all your bank details: credit card number, expiration date, ccv code on the back of the card.

Have a nice day
